src/hooks/appSetting.ts

```diff
@@ -7,19 +7,31 @@ import { getApiHost, getData, getDataOrThrow } from '../config/utils';
 import { QueryClientConfig } from '../types';
 import { convertJs } from '@graasp/sdk';
 import { AppSettingRecord } from '@graasp/sdk/frontend';
+import { WebsocketClient } from '../ws/ws-client';
+import { configureWsAppSettingHooks } from '../ws/hooks/app';
 
-export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
+export default (
+  queryClient: QueryClient,
+  queryConfig: QueryClientConfig,
+  websocketClient?: WebsocketClient,
+) => {
   const { retry, cacheTime, staleTime } = queryConfig;
   const defaultOptions = {
     retry,
     cacheTime,
     staleTime,
   };
+  const { useAppSettingsUpdates } = configureWsAppSettingHooks(websocketClient);
   return {
-    useAppSettings: () => {
+    useAppSettings: (options?: { getUpdates: boolean }) => {
+      const { getUpdates } = options || { getUpdates: true };
```

Comment:
Same here (it works now because there's only one option argument). But for futureproofing, if you later add more arguments:

```suggestion
      const getUpdates = options?.getUpdates ?? true;
```